src/com/serotonin/mango/web/dwr/BaseDwr.java

```diff
@@ -93,8 +89,7 @@ protected PointValueTime prepareBasePointState(String componentId, BasePointStat
         model.put("componentId", componentId);
         model.put("point", pointVO);
         model.put("pointRT", point);
-        model.put(MODEL_ATTR_RESOURCE_BUNDLE, getResourceBundle());
```

Comment:
getResourceBundle()

Check if this method is used somewhere, if not, set the annotation  `@Deprecated(since = "2.7.8")`